### PR TITLE
enhance the webservice flavor to be more useful

### PR DESCRIPTION
**Pull request by jostillmanns**

### CODE REVIEW

File: examples/pligos/flavors/webservice/templates/_container.tpl [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    {{- end }}
+  {{- end }}
+{{- end }}
+
+{{- if .c.command }}
+command:
+  {{- if eq "script" .c.command.type }}
+{{ toYaml .c.command.interpreter | indent 2 -}}
+  - |
+{{ .root.Files.Get (printf "script/%s" .c.command.script) | indent 4 }}
```
Comment (jostillmanns) [inline examples/pligos/flavors/webservice/templates/_container.tpl:21]: use same implementation as for pull secrets

File: examples/pligos/flavors/webservice/templates/registry.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{{- if .Values.pullSecrets -}}
+{{- $root := . -}}
+{{- range .Values.pullSecrets }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "fullname" $root }}-{{ .name }}
+  creationTimestamp: null
```
Comment (jostillmanns) [inline examples/pligos/flavors/webservice/templates/registry.yaml:8]: fix

**Comment by johnholiver** [comment]
Looks good, but you want to do a commit to fix the comments you did yourself, right?
I'll approve after.

**Comment by johnholiver** [comment]
After our weekly, we decided to close this PR. 
We are aiming to initially support examples/flavors that are quite basic and provide understanding on how to use pligos. We could evolve to have more "best practice" flavors, but this will happen in another sprint.

**Pull request closed without merging**